src/gluonts/model/predictor.py

```diff
@@ -106,25 +106,28 @@ def predict(self, dataset: Dataset, **kwargs) -> Iterator[Forecast]:
         raise NotImplementedError
 
     def serialize(self, path: Path) -> None:
-        try:
-            # serialize Predictor type
-            with (path / "type.txt").open("w") as fp:
-                fp.write(fqname_for(self.__class__))
-        except Exception as e:
```

Comment:
Why do all the try-catch need to be removed?

Reply:
They are swallowing actual error messages and stack traces and replace them by some generic error message.

Reply:
Maybe we can add a test, since the build also runs on a GPU instance...